charts/victoria-metrics-cluster/templates/vminsert-deployment.yaml

```diff
@@ -1,5 +1,9 @@
 {{- if .Values.vminsert.enabled -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
don't need to add `if` please change `apiVersion` and bump version of chart in `Chart.yml`

Reply:
you are right, i changed
it's ok for you ?

Reply:
thank you so much